pySDC/implementations/controller_classes/controller_nonMPI.py

```diff
@@ -490,6 +517,13 @@ def it_check(self, local_MS_running):
         if self.params.use_iteration_estimator:
             self.check_iteration_estimator(local_MS_running)
 
+        self.error_estimator.estimate(local_MS_running)
```

Comment:
Can you think of a reason why this would need to be deactivated? E.g., data type stuff? If so, please add a flag.

Reply:
No, that's all "on the lowest level". You run into problems with data types atm only when storing values for Hot Rod, so that's where the error is raised. This estimate function has exactly the purpose to figure out if something should be estimated right now based on parameters as well as iterations, so I don't think there needs to be a further flag.